core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2690,7 +2690,10 @@ object SparkContext extends Logging {
         setActiveContext(new SparkContext(config))
       } else {
         if (config.getAll.nonEmpty) {
-          logWarning("Using an existing SparkContext; some configuration may not take effect.")
+          logWarning("Using an existing SparkContext; some configuration may not take effect. " +
+            "For how to set these configuration correctly, you can refer to " +
+            s"https://spark.apache.org/docs/${SPARK_VERSION.replace("-SNAPSHOT", "")}" +
+            s"/configuration.html#dynamically-loading-spark-properties.")
```

Comment:
This doesn't match the PR title. So now this PR just improves the error message and doc?